mne/tests/test_source_estimate.py

```diff
@@ -283,8 +229,8 @@ def test_io_stc():
     """Test IO for STC files."""
     tempdir = _TempDir()
     stc = _fake_stc()
-    stc.save(op.join(tempdir, "tmp.stc"))
-    stc2 = read_source_estimate(op.join(tempdir, "tmp.stc"))
+    stc.save(os.path.join(tempdir, "tmp.stc"))
+    stc2 = read_source_estimate(os.path.join(tempdir, "tmp.stc"))
```

Comment:
please avoid cosmetic changes. The diff is already super huge. Can I ask you to undo this?